### DISCUSSION TITLE
On/off of beep sound and vibration.

### DISCUSSION THREAD

**Discussion by ericwalker** [Th3Un1q3/flipp_pomodoro#54] [Ideas]
Hope Pomodoro will be able to choose between beep sound and vibration then it could be used in the office or in the library.

**Comment by Th3Un1q3** [comment]
This is planned. Thank you.

I think of switching between:
- silent
- vibro
- vibro and sound
By clicking let's say button left.

**Comment by Th3Un1q3** [comment]
Hi Eric( @ericwalker ),

I wanted to let you know that there is a better way to adjust it to your use case. Simply visit "Settings" > "LCD and Notifications" and make the necessary changes.

Here's a visual aid to help you navigate to the correct settings page:

<img width="524" alt="image" src="https://github.com/Th3Un1q3/flipp_pomodoro/assets/19877114/c4d76fb0-2058-4de5-bf77-171cec1ea969">

As a result, I am closing this feature request and would like to express my gratitude for your contribution.
